guide/installation-procedure.md

```diff
@@ -137,6 +137,14 @@ or
 gdebi webots_{{ webots.version.major }}.{{ webots.version.minor }}.{{ webots.version.bugfix }}_amd64.deb
 ```
 
+#### Server edition
+
+Webots requires some graphical features that are usually not available by default on a linux server edition and additional packages need to be available to make it work.
+
+Webots needs the X Window system and thus the `xserver-xorg-core` and `libxslt1.1` packages have to be installed. These packages are automatically installed when using the DEB package, but in case of the tarball package the user has to manually install them.
```

Comment:
Note that libxslt is not related with the X Window system, it's an XML-related library. Therefore, we shouldn't mention it here.